gslab_misc/ado/stata.toc

```diff
@@ -0,0 +1,34 @@
+p aivreg                Estimates panel IV models using 2SLS within estimator
```

Comment:
Maybe mention that this is a collection of third part packages and provide attribution in the various .pkg files.